bucket/firedm.json

```
@@ -0,0 +1,34 @@
+{
+    "version": "2022.2.5",
+    "description": "Multi-connections download manager",
+    "homepage": "https://github.com/firedm/FireDM",
+    "license": "LGPL-3.0-only",
+    "url": "https://github.com/firedm/FireDM/releases/download/2022.2.5/FireDM_2022.2.5.zip",
+    "hash": "03CB296C3E54C2CEE80B16E1E91708250A7FEE6D338F9BE88ACF9414429210A0",
+    "extract_dir": "FireDM",
+    "pre_install": [
+        "'downloads.dat', 'thumbnails.dat', 'user_themes.cfg' | ForEach-Object {",
+        "    if (!(Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" | Out-Null }",
```

```suggestion
        "    if (!(Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" -ItemType File | Out-Null }",
```

By default New-Item creates a folder.